### PR TITLE
Mat 4570 e cqm title

### PR DESCRIPTION
**Pull request by sb-cecilialiu**
## MADiE PR

Jira Ticket: [MAT-4570](https://jira.cms.gov/browse/MAT-4570)
(Optional) Related Tickets:

### All Submissions
* [x] This PR has the JIRA linked.
* [x] Required tests are included.
* [x] No extemporaneous files are included (i.e Complied files or testing results).
* [x] This PR is merging into the **correct branch**.
* [ ] All Documentation needed for this PR is Complete (or noted in a TODO or other Ticket).
* [ ] Any breaking changes or failing automations are noted by placing a comment on this PR.

### DevSecOps
If there is a question if this PR has a security or infrastructure impact, please contact the Security or DevOps engineer assigned to this project to discuss it further.

* [x] This PR has NO significant security impact (i.e Changing auth methods, Adding a new user type, Adding a required but vulnerable package).
* [ ] All CDN/Web dependencies are hosted internally (i.e MADiE-Root Repo).

### Reviewers
By Approving this PR you are attesting to the following:

*  Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose.
*  The tests appropriately test the new code, including edge cases.
*  If you have any concerns they are brought up either to the developer assigned, security engineer, or leads.

### CODE REVIEW

File: src/components/editMeasure/measureDetails/measureInformation/MeasureInformation.tsx [hunk 1]
```diff
@@ -284,6 +287,31 @@ export default function MeasureInformation() {
...
+          <br />
+          <DisplayDiv>
+            <span className={classes.asterisk}>*</span>
+            <span tw="mr-2" className={classes.fieldName}>
+              ECQM Abbreviated Title:
+            </span>
+          </DisplayDiv>
+          <div className={formRowGapped}>
+            <TextField
+              placeholder="Enter ECQM Abbreviated Title"
```
Comment (adongare) [inline src/components/editMeasure/measureDetails/measureInformation/MeasureInformation.tsx]: this placeholder is different from create measure modal ecqm placeholder

File: src/components/editMeasure/measureDetails/measureInformation/MeasureInformation.tsx [hunk 2]
```diff
@@ -284,6 +287,31 @@ export default function MeasureInformation() {
...
 
+          <br />
+          <DisplayDiv>
+            <span className={classes.asterisk}>*</span>
+            <span tw="mr-2" className={classes.fieldName}>
+              ECQM Abbreviated Title:
+            </span>
+          </DisplayDiv>
+          <div className={formRowGapped}>
+            <TextField
```
Comment (RohitKandimalla) [inline src/components/editMeasure/measureDetails/measureInformation/MeasureInformation.tsx:265]: Can we use components from madie-design-systems?

**Review by RohitKandimalla — CHANGES_REQUESTED** [review]

File: src/components/editMeasure/measureDetails/measureInformation/MeasureInformation.tsx (hunk 1 above)
Comment (sb-cecilialiu) [reply src/components/editMeasure/measureDetails/measureInformation/MeasureInformation.tsx]: Thanks will change it.

File: src/components/editMeasure/measureDetails/measureInformation/MeasureInformation.tsx (hunk 2 above)
Comment (sb-cecilialiu) [reply src/components/editMeasure/measureDetails/measureInformation/MeasureInformation.tsx:265]: Thanks, will do!

Comment (sb-cecilialiu) [reply src/components/editMeasure/measureDetails/measureInformation/MeasureInformation.tsx:265]: Thanks, will do!

Comment (RohitKandimalla) [reply src/components/editMeasure/measureDetails/measureInformation/MeasureInformation.tsx:265]: Thats awesome, they looks great now

**Review by RohitKandimalla — APPROVED** [review]

**Review by jkotanchik-SB — APPROVED** [review]

File: src/components/editMeasure/measureDetails/measureInformation/MeasureInformation.tsx (hunk 1 above)
Comment (sb-cecilialiu) [reply src/components/editMeasure/measureDetails/measureInformation/MeasureInformation.tsx]: This is done.

**Pull request merged**